R/nm-file.R

```diff
@@ -87,10 +87,12 @@ nm_par_tab <- function(.mod) {
 
 #' @describeIn nm_file Reads the input data file from a `bbi_nonmem_model` or
 #'   `bbi_nonmem_summary` object
+#' @param filter Logical (`T`/`F`). If `TRUE`, filter data based on `IGNORE` and
```

Comment:
I opted to adjust the `.rename` documentation, since we have `"Logical ('T'/'F')"` in a number of other places too. Let me know if you disagree with this move.